pkg/plugins/scms/github/pullrequest.go

```diff
@@ -194,10 +194,11 @@ func (p *PullRequest) CreateAction(title, changelog, pipelineReport string) erro
 	}
 
 	if p.spec.AutoMerge {
-		err = p.EnablePullRequestAutoMerge()
-		if err != nil {
-			return err
-		}
+		logrus.Warningf("GitHub automerge temporarly disabled on pullrequest due to https://github.com/updatecli/updatecli/issues/1252")
```

Comment:
super minor NIT:

```suggestion
		logrus.Warningf("GitHub automerge temporarily disabled on pull request due to https://github.com/updatecli/updatecli/issues/1252")
```

Reply:
Thanks, I open this pullrequest a bit quickly :)